sdk/storage/storage-queue/src/utils/utils.common.ts

```diff
@@ -114,6 +114,28 @@ export function getURLQueries(url: string): { [key: string]: string } {
   return queries;
 }
 
+function getDevConnString(connectionString: string): string {
+  // Development Connection String
+  // https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string#connect-to-the-emulator-account-using-the-well-known-account-name-and-key
+  const partialConnectionString = `DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;
+        AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;`;
```

Comment:
The string is from the [DOCS](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string#connect-to-the-emulator-account-using-the-well-known-account-name-and-key)

Hard coding is fine?

Reply:
Hard coding is fine, but better move to constants.ts

Reply:
Moved